code/modules/spells/spell_types/invoked/create_wise_trees.dm

```diff
@@ -1,10 +1,10 @@
 /obj/effect/proc_holder/spell/invoked/transform_tree
```

Comment:
This spell doesn't actually remove the branches, take a look at the tree felling in newtree.dm

Reply:
Fixed.